src/passes/PostEmscripten.cpp

```diff
@@ -60,6 +64,29 @@ struct PostEmscripten : public WalkerPass<PostWalker<PostEmscripten>> {
   }
 };
 
+} // namespace
+
+struct PostEmscripten : public Pass {
+  void run(PassRunner* runner, Module* module) override {
+    // Apply the sbrk ptr, if it was provided.
+    auto sbrkPtrStr =
+      runner->options.getArgumentOrDefault("emscripten-sbrk-ptr", "");
+    if (sbrkPtrStr != "") {
+      auto sbrkPtr = std::stoi(sbrkPtrStr);
+      ImportInfo imports(*module);
+      auto* func = imports.getImportedFunction(ENV, "emscripten_get_sbrk_ptr");
+      if (func) {
+        Builder builder(*module);
+        func->body = builder.makeConst(Literal(int32_t(sbrkPtr)));
+        func->module = func->base = Name();
+      }
+    }
+
+    // Optimize calls
+    OptimizeCalls().run(runner, module);
```

Comment:
This feels awkward, but I'm not sure if there's a better way to replace sbrk keeping the order before this

Reply:
Yeah. I think some of the fastcomp-specific parts can be removed when we remove fastcomp, and then this pass can get simpler again.